CHANGES.rst

```diff
@@ -2,11 +2,18 @@
  CHANGES
 =========
 
-4.0.2 (unreleased)
+4.1.0 (unreleased)
 ==================
 
-- Nothing changed yet.
+- The help namespace no longer modifies the global help object on
+  traversal. Instead it returns a new proxy object. This makes it
+  thread-safe. See `issue 4
+  <https://github.com/zopefoundation/zope.app.onlinehelp/issues/4>`_.
 
+- ``getTopicFor`` now really returns the first found topic in the
+  event that the object implements multiple interfaces that have
+  registered topics for the given view. Previously it would return the
+  topic for the least-specific interface.
```

Comment:
I think adding the test case made that commit obsolete. Nothing seemed to rely on the broken behaviour because nothing registers multiple interfaces and topics for the same view; I can't think of why someone would do that.